packages/widgetbook/lib/src/settings_panel/settings_panel.dart

```diff
@@ -1,20 +1,38 @@
 import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:widgetbook/src/addons/addon_provider.dart';
 import 'package:widgetbook/src/constants/radii.dart';
-import 'package:widgetbook/src/knobs/knobs_panel.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_core/widgetbook_core.dart' as core;
 
 class SettingsPanel extends StatelessWidget {
   const SettingsPanel({
-    Key? key,
-  }) : super(key: key);
+    super.key,
+  });
 
   @override
   Widget build(BuildContext context) {
+    final addons = context.watch<AddOnProvider>().value;
+    final knobs = context.watch<KnobsNotifier>().all();
     return DecoratedBox(
       decoration: BoxDecoration(
-        color: Theme.of(context).colorScheme.surface,
+        color: Theme.of(context).colorScheme.inversePrimary.withOpacity(0.05),
```

Comment:
The `surface` color looks more like the design, but its hex code is not the same.
The `inversePrimary` just matches the name of the color in the design but not its hex code 😅
I feel like the `inversePrimary` looks too blue and different than the design, and I remember this happening some place else, do you think the colors need to be fixed?